docs/book/src/SUMMARY.md

```diff
@@ -19,16 +19,25 @@
   - [What's in a controller?](./cronjob-tutorial/controller-overview.md)
   - [Implementing a controller](./cronjob-tutorial/controller-implementation.md)
 
-      - [You said something about main?](./cronjob-tutorial/main-revisited.md)
+    - [You said something about main?](./cronjob-tutorial/main-revisited.md)
 
+  - [Implementing webhook](./cronjob-tutorial/webhook-implementation.md)
   - [Running and deploying the controller](./cronjob-tutorial/running.md)
+  
+    - [Deploying the cert manager](./cronjob-tutorial/cert-manager.md)
+    - [Deploying webhooks](./cronjob-tutorial/running-webhook.md)
+    - [Kind cluster](./cronjob-tutorial/kind.md)
```

Comment:
This needs to either go first, or go in the reference documentation, or something -- it feels awkward here.